### ISSUE TITLE
Update existing playlist from platform

### ISSUE THREAD

**Issue by Freelance-Bum** [futo-org/grayjay-android#1352]
### Use Cases

My use case here is with the YouTube plugin, but other platforms may benefit as well if my proposal is implemented. In YouTube, you can add someone else's playlist to your list of saved playlists. If the maintainer of that playlist adds a new video, you will still see the update in your playlists in your own profile. Currently, the only way to update that playlist is to reimport, which then creates a whole duplicate playlist and uses a bunch of calls

### Proposal

Have a button in the playlist that makes grayjay check for videos that don't already exist in the local playlist copy that are present in the platform playlist, then add those missing videos to the local copy. (However this can be done most efficiently is better, obviously. My suggestion may not be the most efficient)

Another idea is that you could have a system that runs periodic checks (like the subscriptions do) instead of manually pressing a button.

### References

I tried to find anyone else who suggested this, since it seemed like more people would have this annoyance than just me, but I couldn't find any. I apologize if there was one I missed.

**Comment by Scoots1994** [comment]
I would love this.  Right now, because I download playlists I end up having multiple copies of each playlist and have to stop and start several times to get the new additions to download, and sometimes end up with duplicate downloads

**Comment by nastynick64** [comment]
Agreed.This could be very useful. For example: I have a playlist of a tutorial series that is ongoing, the guy making the tutorial does a great job, but, i'm not interested in most of the other things he makes tutroials about. So it would be nice, if whenever he adds a new video to this playlist, it would update, so I could follow along.

**Comment by TheNewsYorkTime** [comment]
I have a lot of playlist saved, and this would be very useful for avoiding duplicates. In addition this would also help my sync my YouTube and grayjay watching. For instance if I add a video to my YouTube watcb later, then grayjay could find that and add it to its version of the playlist. Of course there is an issue when I remove a video on grayjay. That video isn't removed from the youtube playlist, and therefore it would get readded. That's kind of a problem. Also if I add a video. What happens then?
